packages/react/src/components/Toolbar/ToolbarItem.tsx

```diff
@@ -135,6 +161,31 @@ class ToolbarItem extends UIComponent<WithAsProp<ToolbarItemProps>, ToolbarItemS
       </ElementType>
     )
 
+    const submenu = menuOpen ? (
```

Comment:
nit: I would keep consistency as we have across our lib, to have it as a function `renderSubmenu = () => { ... }`